contrib/kafka/pkg/reconciler/kafkasource_test.go

```diff
@@ -65,7 +65,7 @@ func init() {
 	v1.AddToScheme(scheme.Scheme)
 	corev1.AddToScheme(scheme.Scheme)
 	sourcesv1alpha1.SchemeBuilder.AddToScheme(scheme.Scheme)
-	genericv1alpha1.SchemeBuilder.AddToScheme(scheme.Scheme)
+	eventingsourcesv1alpha1.SchemeBuilder.AddToScheme(scheme.Scheme)
```

Comment:
why not just use aws here and kafka in aws so we don't have to pull in eventingsources, or is there another reason for it?

Reply:
it needs ContainerSource from eventing